libs/guid/tests/oguidtest.cpp

```diff
@@ -0,0 +1,196 @@
+#include "motifcpp/motifcpptest.h"
+#include <guid/comparison.h>
+#include <guid/OGuid.h>
+#include <optional>
+
+// {7714B323-69D2-4B37-B6A1-1B433E6EA023}
+const GUID nonNullTestGuid = {0x7714b323, 0x69d2, 0x4b37, {0xb6, 0xa1, 0x1b, 0x43, 0x3e, 0x6e, 0xa0, 0x23}};

```

Comment:
>nonNullTestGuid [](http://example.com/codeflow?start=11&length=15)

All types and variables declared in .cpp files must be either static or use an anonymous namespace.